### PR TITLE
add an option to change input-icon

### PR DESCRIPTION
**Pull request by yihou**
cater for Vuetify stack that uses custom icons.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,10 @@ If you do not want a presets list you can use this boolean and you'll only see t
 
 Switches the list of presets and the datepickers into dark mode.
 
+#### `input-icon` : String
```
Comment (praveenpuglia) [inline README.md]:
```suggestion
#### `prepend-icon` : String
```

File: src/DateRange.vue [hunk 2]
```diff
@@ -25,7 +25,7 @@
           :label="`${labels.start}(${format})`"
           name="startDate"
           class="date-range__pickers-input"
-          prepend-icon="event"
+          :prepend-icon="inputIcon"
```
Comment (praveenpuglia) [inline src/DateRange.vue]:
```suggestion
          :prepend-icon="prependIcon"
```

File: src/DateRange.vue [hunk 3]
```diff
@@ -88,6 +88,10 @@ export default {
       type: Boolean,
       default: false,
     },
+    inputIcon: {
```
Comment (praveenpuglia) [inline src/DateRange.vue]:
```suggestion
    prependIcon: {
```

**Comment by yihou** [comment]
ahh, you are right, will make the changes. thanks

**Comment by praveenpuglia** [comment]
Thanks so much! :)

**Pull request merged**